fix for php7

darwin26 opened the pull request:
update fix for php7

fixes #9 
fixes #10

darwin26 committed:
Update rex_nav_simple_html_dom.php

more fixes for php7.3.x

darwin26 committed:
Merge pull request #1 from darwin26/php7fix-update-patch-1

Update rex_nav_simple_html_dom.php

schuer reviewed (approved):
Approve, ungetestet.

schuer commented:
@darwin26 Das AddOn ist zur Zeit ohne Maintainer. Hättest du Lust, dich drum zu kümmern, den PR zu mergen und ein neues Release zu veröffentlichen?

schuer commented:
Und bei der Gelegenheit vielleicht noch den Vorschlag aus #8 mit aufnehmen :)

darwin26 commented:
yoa ... (du wirst es nciht glauben) aber ich hab keinen Plan wie was wo (mit git) ...
ich übernehme es gerne. und mergen sollte nicht so tragisch sein ;) ...

webregie commented:
Keinen Plan? So ganz funktioniert das Ding mit 7.3 und 7.4 noch nicht. Besonders die 2. Ebene. Ich muss das auch mal untersuchen. Aber vielleicht hat ja jemand schon gefummelt. 
Danke für die Reaktionen übrigens!

darwin26 committed:
listItemClassFromMetaField (fix #8)

fix von fietstouring
Damit das auch mit multiple Klassen funktioniert habe ich in rex_nav.php Zeile 207 durch folgendes ausgetauscht:
$cssClasses .= ' ' . trim(str_replace("|", " ", $cat->getValue($this->listItemClassFromMetaField)));

darwin26 committed:
Merge pull request #2 from darwin26/darwin26-patch-fix#8

listItemClassFromMetaField (fix #8)

darwin26 commented:
"(webregie ) Keinen Plan? " ... 
das mit dem git und so ... i do my best ;) ...

schuer commented:
@darwin26 Hab dir im Slack geschrieben.
